source/PlayerInfo.cpp

```diff
@@ -3475,6 +3475,20 @@ void PlayerInfo::RegisterDerivedConditions()
 		return retVal;
 	});
 
+	// Check if an attribute is present in fleet.
+	auto &&shipAttributeProvider = conditions.GetProviderPrefixed("ship attribute: ");
+	auto shipAttributeFun = [this](const string &name) -> int64_t
+	{
+		auto attrib = name.substr(strlen("ship attribute: "));
+		int64_t sum = 0;
+		for(const shared_ptr<Ship> &ship : ships)
+			sum += ship->Attributes().Get(attrib);
+
```

Comment:
```suggestion
```
This is more in line with the functions above. I'm wondering if the variable should be renamed to `retVal` for consistency as well, I'm not sure.